### DISCUSSION TITLE
where is rtmpmuxer

### DISCUSSION THREAD

**Discussion by emirbeytekin** [HaishinKit/HaishinKit.swift#1436] [Q&A] [answered]
<img width="1444" alt="image" src="https://github.com/shogo4405/HaishinKit.swift/assets/13369070/f2418f5b-de6d-400d-b52f-b2d861c7c761">
Module 'HaishinKit' has no member named 'RTMPMuxer'

**Comment by shogo4405** [comment, marked as answer]
No. RTMPMuxer is internal class. so you can't acccess.
